app/bundles/CoreBundle/Assets/js/libraries/froala/plugins/token.js

```diff
@@ -52,12 +52,12 @@
                     if (k.match(/assetlink=/i) && v.match(/a:/)){
                         delete tokens[k];
                         var nv = v.replace('a:', '');
-                        k = '<a title=\'Asset Link\' href=\'' + k + '\'>' + nv + '</a>';
+                        k = '<a title="Asset Link" href="' + k + '">' + nv + '</a>';
                         tokens[k] = nv;
                     } else if (k.match(/pagelink=/i) && v.match(/a:/)){
                         delete tokens[k];
                         nv = v.replace('a:', '');
-                        k = '<a title=\'Page Link\' href=\'' + k + '\'>' + nv + '</a>';
+                        k = '<a title="Page Link" href=""' + k + '">' + nv + '</a>';
                         tokens[k] = nv;
                     } else if (k.match(/dwc=/i)){
                         var tn = k.substr(5, k.length - 6);
```

Comment:
on line https://github.com/mautic/mautic/blob/2670a28bf1be926c56c8bd37342205923d1ea1e2/app/bundles/CoreBundle/Assets/js/libraries/froala/plugins/token.js#L94 

you have to change by 
```
var newOption = '<li role="presentation"><a class="fr-command" tabIndex="-1" role="option" data-cmd="token" data-param1="' + encodeURI(val) + '" title="' + title + '">' + title + badge + '</a></li>';
```

top encode the value of "val" to prevent bug into this screen :  
![image](https://user-images.githubusercontent.com/14074272/66813949-086acc00-ef36-11e9-9376-7e8a16c62b2f.png)

Reply:
Nice catch. Thanks